### ISSUE TITLE
[OS X] [General] Hint in build documentation for the QT Version

### ISSUE THREAD

**Issue by pierrejochem** [owncloud/client#5129]
There should be a hint in the build introductions that it will not work with QT Version 5.6 or higher due to missing qtwebkit! If a user is running brew updates then this could happen...

https://doc.owncloud.org/desktop/2.0/building.html#mac-os-x

This is not working:

```
brew tap owncloud/owncloud
brew install $(brew deps owncloud-client)
```

always getting qt 5.6 :-(

Anybody a clue? @guruz @danimo @ogoffart ?

**Comment by pierrejochem** [comment]
https://doc.owncloud.org/desktop/2.0/building.html#mac-os-x

This is not working:

```
brew tap owncloud/owncloud
brew install $(brew deps owncloud-client)
```

**Comment by pierrejochem** [comment]
Anybody a clue?

**Comment by dragotin** [comment]
AFAIK there is qtwebkit for Qt5.6 in a compatibility package.

**Comment by pierrejochem** [comment]
Okay, I will search for it. The only fact I found is that it is remove because it was depricated.

**Comment by SamuAlfageme** [comment]
Addressed in #5471
